ingestion/src/metadata/ingestion/source/database/redshift_lineage.py

```diff
@@ -42,7 +42,7 @@ class RedshiftLineageSource(RedshiftQueryParserSource, LineageSource):
     filters = """
         AND (
           querytxt ILIKE 'create table %% as select %%'
-          OR querytxt ILIKE 'insert %%'
+          OR querytxt ILIKE 'insert%%'
```

Comment:
I think for insert we should make it as `%%insert%%` in case the insert statement is followed by a comment.

same goes for create statements as well also the issue regarding redshift was some queries like `create table as (select ....)` were not getting caught in this so we need to cover that as well